config.lua

```diff
@@ -547,6 +547,18 @@ lspconfig.sumneko_lua = add_lsp {
   }
 }
 
+---# LaTeX Texlab language server
+--- __Status__: Works
+--- __Site__: https://github.com/latex-lsp/texlab
+--- __Installation__: git clone https://github.com/latex-lsp/texlab.git , then inside the texlab folder, run: cargo build --release
+--- __Note__: Rust has to be installed
+lspconfig.texlab = add_lsp {
+  name = "texlab",
+  language = "latex",
+  file_patterns = { "%.tex$", "%.bib$" },
```

Comment:
Would it make sense to also add the file formats specified here https://github.com/lite-xl/lite-xl-plugins/blob/3eb5ef21dcb560cf41e84f7cba37e957e5138502/plugins/language_tex.lua#L6 ?

Reply:
Yes, it does make sense indeed. I will add them.